week-1/implement/sum.js

```diff
@@ -0,0 +1,18 @@
+function sumNumbers(numbers) {
+    let totalSum = 0;
+  
+    for (const element of numbers) {
+      if (isNumeric(element)) {
+        totalSum += element;
+      }
+    }
+  
+    return totalSum;
+  }
+  
+  function isNumeric(value) {
+    return typeof value === 'number' && !isNaN(value);
+  }
+  
```

Comment:
again you don't really need to create(isNumeric) function for only one line, you can just copy the body of the function to the if statement, that makes it easy to read your code!